client/lclusterc.go

```diff
@@ -0,0 +1,324 @@
+
+	// Safety check, ensure no errors have occurred.
+	if err != nil {
+		fmt.Printf("checkJobOnServer() --> unable to connect to grpc " +
+			"server at address")
+		return
+	}
+
+	// Defer the connection for the time being, but eventually it will be
+	// closed once we've finished with it.
+	defer connection.Close()
+
+	// Create an lcluster client
+	lcluster_client := pb.NewLclusterdClient(connection)
+
+	// Assemble a check job request object
+	request := &pb.CheckJobRequest{Pid: pid}
+
+	// Grab the current background context.
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+
+	// Using the check job request defined above, go ahead and attempt to
+	// stop the job
+	response, err := lcluster_client.CheckJob(ctx, request)
+
+	// Cancel the current context since this has either generated a
+	// response or an error.
+	cancel()
+
+	// Safety check, ensure that no error has occurred.
+	if err != nil {
+		fmt.Printf("checkJobOnServer() --> the following error has " +
+			"occurred\n" + err.Error())
+	}
+
+	// If an internal server-side error has occurred...
+	if response.Rc == CJR_CORRUPTED_SERVER_INPUT {
+		fmt.Printf("An internal server-side error has occurred.\n")
+		return
+
+		// Unknown job state
+	} else if response.Rc == CJR_UNKNOWN {
+		fmt.Printf("The job '" + uuid + "' appears to have an unknown " +
+			"status.\nConsider contacting the server operator.\n")
+		return
+
+		// Job does not exist
+	} else if response.Rc == CJR_PROCESS_NOT_EXIST {
+		fmt.Printf("The job '" + uuid + "' is not present.\n")
+		return
+
+		// Job is queued
+	} else if response.Rc == CJR_PROCESS_QUEUED {
+		fmt.Printf("The job '" + uuid + "' is queued.\n")
+		return
+
+		// Job is currently running
+	} else if response.Rc == CJR_PROCESS_ACTIVE {
+		fmt.Printf("The job '" + uuid + "' is currently active.\n")
+		return
+	}
+
+	// otherwise if none of the above happened, print out a message
+	// stating this is unknown, with the response code
+	fmt.Printf("The following undefined response code was given back: " +
+		strconv.FormatInt(response.Rc, 10) + "\n")
+	fmt.Printf("Consider contacting the server operator.\n")
+}
+
+//! Remove a job from the server.
+/*
+ * @param    string    uuid of job to remove
+ *
+ * @return   none
+ */
+func removeJobFromServer(uuid string) {
+
+	// Input validation
+	if len(uuid) < 1 {
+		fmt.Printf("removeJobFromServer() --> invalid input")
+		return
+	}
+
+	// attempt to cast the pid into an int since that is StopJobRequest
+	// uses as the Pid var type
+	pid, err := strconv.ParseInt(uuid, 10, 64)
+
+	// if an error occurs, print it out
+	if err != nil || pid < 1 {
+		fmt.Printf("Please enter a valid int64 value.\n")
+		return
+	}
+
+	// Dial a connection to the grpc server.
+	connection, err := grpc.Dial(grpcServerAddr+grpcPort,
+		grpc.WithInsecure())
+
+	// Safety check, ensure no errors have occurred.
+	if err != nil {
+		fmt.Printf("removeJobFromServer() --> unable to connect to grpc server at address")
```

Comment:
Done, using the standard golang "log" package; it seems to work as a drop-in replacement and eliminates the purpose for my stdlog function.

https://github.com/rbisewski/lclusterd/commit/a517d6bdc2057a8e1301bc12d15ecfe2e3e13991